src/main/java/knightminer/simplytea/core/Registration.java

```diff
@@ -213,7 +216,9 @@ static void registerObjects(final RegisterEvent event) {
       cup_tea_chai = register(r, new TeaCupItem(props.food(Config.SERVER.chai_tea)), "cup_tea_chai");
       cup_tea_iced = register(r, new TeaCupItem(props.food(Config.SERVER.iced_tea)), "cup_tea_iced");
       cup_tea_chorus = register(r, new TeaCupItem(props.food(Config.SERVER.chorus_tea)), "cup_tea_chorus");
+      cup_water_hot = register(r, new Item(props.food(new FoodProperties.Builder().alwaysEat().build())), "cup_water_hot");
       cup_cocoa = register(r, new CocoaItem(props.food(Config.SERVER.cocoa)), "cup_cocoa");
+      cup_frothed = register(r, new CocoaItem(props.food(Config.SERVER.cocoa)), "cup_frothed");
```

Comment:
this should be `frothed_milk`, not `cocoa`. Also, like I suggested in `CocoaItem`, lets pass in `Config.SERVER.cocoa`/`Config.SERVER.frothed_milk` to the constructor parameter so we have cure effect properties.

Reply:
I thought I had already caught and changed that. Maybe I didn't push in the change. It's fixed now.